### ISSUE TITLE
Version 3.6.0 in npm but 3.5.4 in master package.json.

### ISSUE THREAD

**Issue by asaarnak** [wbuchwalter/tslint-loader#111]
Little confusing when npm has newer version than master branch package.json file.

**Comment by bhovhannes** [comment]
https://www.npmjs.com/package/tslint-loader shows 3.5.4 as latest version.
How did you see version 3.6.0 ?

**Comment by asaarnak** [comment]
In versions tab.

**Comment by asaarnak** [comment]
![screenshot from 2018-12-18 09-10-01](https://user-images.githubusercontent.com/808420/50137734-c71d7780-02a4-11e9-8583-fd32848cda11.png)

**Comment by bhovhannes** [comment]
Perhaps it was published in `@next` release channel. The last stable version is `3.5.4`.

**Comment by asaarnak** [comment]
On picture it shows `10 months ago`
Seems like `3.6.1` should have been released instead of `3.5.4`

**Comment by tommck** [comment]
@bhovhannes Can you deprecate the 3.6.0 package then?  It is the highest number out there and was published _before_ 3.5.4

**Comment by bhovhannes** [comment]
Yeah, TSLint is deprecated, and `npm install tslint-loader` installs version 3.5.4 of tslint-loader which matches with what we have in the github repo.  
So, marking 3.6.0 as deprecated won't harm anyone.

https://www.npmjs.com/package/tslint-loader/v/3.6.0
